packages/ctl/ctl.js

```diff
@@ -34,6 +36,48 @@ function print(...content) {
 	console.log(...content);
 }
 
+async function getEditor(argsEditor) {
+	let editor = "";
+	if (argsEditor) {
+		editor = argsEditor;
+		/* eslint-disable */
+	} else if (process.env.EDITOR) {
+		editor = process.env.EDITOR;
+	} else if (process.env.VISUAL) {
+		editor = process.env.VISUAL
+		/* eslint-enable */
+		// needed for the process.env statements to not be flagged by eslint
+		// prio for editors is CLI arg > env.EDITOR > env.VISUAL
+	} else {
+		editor = -1;
+	}
+	return editor;
+}
```

Comment:
This entire function could be

```js
function getEditor(argsEditor) {
    return argsEditor || process.env.EDITOR || process.env.VISUAL || -1;
}
```

Only reason I could see to write it as a larger async function would  be if you did some more testing to check the validity of the various editors, like checking that they are available in the path, but you don't.